api/tests/command_test.py

```diff
@@ -0,0 +1,13 @@
+from django.core.management import call_command
```

Comment:
good call i did not think to make the test break to see if it is working; the fact that it is running and the tests are still as fast as they are means the import is much faster than i was expecting, so my concern is probably unfounded


since it is so fast i agree with you we might as well leave the whole import in there in order to  have everything covered